src/pages/[language]/lustrum/index.vue

```diff
@@ -71,23 +71,17 @@
       <h2 class="page-lustrum__title-blogs h3">
         {{ data.page.blogsTitle }}
       </h2>
```

Comment:
Adding the title will means we can't reuse it on the blog and home pages, unless we make the title optional but then when there is no title the `section` tag might not make sense.
What would you prefer?

Reply:
There is a title on the home page and on the blog page too right? I'm probably misunderstanding.

![image](https://github.com/voorhoede/voorhoede-website/assets/5701149/ed31a562-1f5c-4971-a86a-3d1eb5a2f733)
![image](https://github.com/voorhoede/voorhoede-website/assets/5701149/174f844d-53be-4ff0-a063-2a14749222b8)


They do look wildly different though...

Reply:
That's a description not a title, there is a sr only title. But because of the very different looks I think it will be hard to get a section working on the other pages. 
I can make a `blogs-section` component/block with a title and list and use that on the landing page, then on the code use the `blogs-list` component on the other pages, and on the `blogs-section`?

Reply:
I have done this for the events https://github.com/voorhoede/voorhoede-website/pull/646 there all the sections look the same

Reply:
Ah okay gotcha, I'll leave the call to split this into globs-section and blogs-list to you, it might not be worth it? Feel free to merge.

Reply:
I decided to split it, mainly for consistency on Dato, since both the events and blogs sections are the same to me it makes sense that they have the same structure in common.